xbmc/Application.cpp

```diff
@@ -398,6 +399,7 @@ bool CApplication::Create(const CAppParamParser &params)
 #ifdef HAS_PYTHON
   SERVICES::CServiceManager::GetInstance().RegisterService(std::shared_ptr<XBPython>(new XBPython), 0);
 #endif
+  SERVICES::CServiceManager::GetInstance().RegisterService(std::shared_ptr<PLAYLIST::CPlayListPlayer>(new PLAYLIST::CPlayListPlayer));
```

Comment:
should we use `std::make_shared` instead where possible?

Reply:
sure.